com.pinegrow.Pinegrow.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.pinegrow.Pinegrow</id>
+	<metadata_license>CC-BY-SA</metadata_license>
+	<name>Pinegrow</name>
+	<summary>Visual web editor for professionals</summary>
+  <description>
+  <p>
+  Pinegrow lets you work faster with HTML, CSS / SASS, Bootstrap, Tailwind CSS, GreenSock Interactions, WordPress and WooCommerce.
+  
```

Comment:
there should be more `<p>` here to mark the paragraphs.

Reply:
> there should be more `<p>` here to mark the paragraphs.

Fixed